OOStepByStep/Person.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOStepByStep
+{
+    public class Person
+    {
+        public string Name { get; set; }
```

Comment:
person info like name and age could be private, since other should not directly modify person's name and age